### ISSUE TITLE
Config by attribute is not working: ex. Restrict by dateMin

### ISSUE THREAD

**Issue by numediaweb** [Eonasdan/tempus-dominus#2881]
### Prerequisites

- [X] I have [searched](https://github.com/Eonasdan/tempus-dominus/issues?q=is%3Aissue) for duplicate or closed issues.
- [X] I have validated my setup against the latest version on [StackBlitz](https://stackblitz.com/edit/tempus-dominus-v6-simple-setup).
- [X] I have read the [contributing guidelines](https://github.com/Eonasdan/tempus-dominus/blob/master/.github/CONTRIBUTING.md).

### Describe the issue

I have checked the docs but couldn't find any mention of providing "inline" config options via the input element data attributes.
I tried every combination to name the attributes but none of them work:
```html
 <input id="datetimepicker1Input" type="text" class="form-control"
        data-td-target="#datetimepicker1" 
        data-td-restrictions-min-date="12/01/2023 00:00 A" 
        data-td-restrictions-mindate= "12/01/2023 00:00 A"
        data-td-restrictions-minDate="12/01/2023 00:00 A" 
>
```
Am I missing some parts of the docs?

Any help would be appreciated :) 

### StackBlitz fork

https://stackblitz.com/edit/tempus-dominus-v6-repl-zrh1cx?file=index.html,example.js

### What operating system(s) are you seeing the problem on?

macOS

### What browser(s) are you seeing the problem on?

Chrome

### What version of are you using? You can find this information from the sample StackBlitz.

v6

### What your browser's locale? You can find this information from the sample StackBlitz.

en-US

**Comment by Eonasdan** [comment]
Hi. I'll push out a fix for this soon. You will want to use `data-td-restrictions_mindate`. JS converts the `data-` attributes to `CamelCase` and strips the `-` marks. Should go out this week

**Comment by Eonasdan** [comment]
This is fixed in 6.9.4
